src/SignalR/clients/csharp/Client/test/UnitTests/HubConnectionBuilderTests.cs

```diff
@@ -77,4 +77,30 @@ public void AddMessagePackProtocolSetsHubProtocolToMsgPack()
 
         Assert.IsType<MessagePackHubProtocol>(serviceProvider.GetService<IHubProtocol>());
     }
+
+    [Fact]
```

Comment:
Can we add a test that configures both (use different values for each)? I want to double check that calling `Configure` on the same options type multiple times works.